tools/changelog-helper.sh

```diff
@@ -39,6 +39,8 @@ export GH_REPO=jamulussoftware/jamulus
 
 PR_LIST_LIMIT=300
 TRANSLATION_ENTRY_TEXT="GUI: Translations have been updated:"
+TRANSLATION_ENTRY_TEXT2="Translations updated from Hosted Weblate"
+BUNDLED_QT_UPDATE_TEXT="Build: Updated bundled Qt6"
```

Comment:
There's a bunch of these defined in the [bump dependencies](https://github.com/jamulussoftware/jamulus/blob/main/.github/workflows/bump-dependencies.yml#L66) workflow - it's probably worth covering all of them (separate groups).

Reply:
OK, done